swap_meet/vendor.py

```diff
@@ -1,2 +1,98 @@
 class Vendor:
-    pass
\ No newline at end of file
+    MAX_AGE = 1000
```

Comment:
What would happen if we had a dinosaur fossil that was more than a thousand years old? To accommodate this case, we can set our max age to infinity using `float('inf')`. We can also consider doing this in the method itself instead of having a constant attached to our `Vendor`. The constant is just an implementation detail that helps us find the youngest object, it's not something that we need to expose to other parts of out code. I really like the idea of using a constant though!